pandas/core/generic.py

```diff
@@ -2897,6 +2897,17 @@ def to_latex(
         if multirow is None:
             multirow = config.get_option("display.latex.multirow")
 
+        if is_list_like(formatters) and not isinstance(formatters, dict):
+            formatter_elems_is_str = all(isinstance(elem, str) for elem in formatters)
+            formatter_elems_is_lambda = all(callable(elem) for elem in formatters)
+            if formatter_elems_is_str:
+                formatters = [
+                    (lambda style: lambda x: "{0:{1}}".format(x, style))(style)
+                    for style in formatters
+                ]
+            elif not formatter_elems_is_lambda:
+                raise ValueError("Formatters elements should be strings")
```

Comment:
So if a user passes a mix of callable and strings this error gets thrown no? Is there a way to more generically support that case?